lldb/source/Target/SwiftLanguageRuntimeNames.cpp

```diff
@@ -182,34 +188,29 @@ static ThunkAction GetThunkAction(ThunkKind kind) {
 class ThreadPlanStepInAsync : public ThreadPlan {
 public:
   static bool NeedsStep(SymbolContext &sc) {
-    if (sc.line_entry.IsValid() && sc.line_entry.line == 0) {
+    if (sc.line_entry.IsValid() && sc.line_entry.line == 0)
       // Compiler generated function, need to step in.
       return true;
-    }
 
     // TEMPORARY HACK WORKAROUND
-    if (!sc.symbol || !sc.comp_unit) {
+    if (!sc.symbol || !sc.comp_unit)
       return false;
-    }
     auto fn_start = sc.symbol->GetFileAddress();
     auto fn_end = sc.symbol->GetFileAddress() + sc.symbol->GetByteSize();
     int line_entry_count = 0;
-    if (auto line_table = sc.comp_unit->GetLineTable()) {
+    if (auto *line_table = sc.comp_unit->GetLineTable()) {
       for (uint32_t i = 0; i < line_table->GetSize(); ++i) {
         LineEntry line_entry;
         if (line_table->GetLineEntryAtIndex(i, line_entry)) {
-          if (!line_entry.IsValid() || line_entry.line == 0) {
+          if (!line_entry.IsValid() || line_entry.line == 0)
             continue;
-          }
 
           auto line_start = line_entry.range.GetBaseAddress().GetFileAddress();
-          if (line_start >= fn_start && line_start < fn_end) {
-            if (++line_entry_count > 1) {
+          if (line_start >= fn_start && line_start < fn_end)
```

Comment:
In general it's better not to mix this sort of cleanup into a big patch like this.

Reply:
my bad, Adrian had commented on the use of curly braces so I did a pass on all of the code I had added since there was a bunch. It could also have been something I did earlier before the size had increased.